internal/profile/profile_json.go

```diff
@@ -10,46 +10,39 @@ import (
 	"fmt"
 	"io"
 	"os"
-	"os/user"
+	"strconv"
 	"time"
 
 	"github.com/elastic/go-resource"
-
-	"github.com/elastic/elastic-package/internal/version"
 )
 
 // Metadata stores the data associated with a given profile
 type Metadata struct {
 	Name        string    `json:"name"`
 	DateCreated time.Time `json:"date_created"`
-	User        string    `json:"user"`
 	Version     string    `json:"version"`
-	Path        string    `json:"path"`
```

Comment:
User and Path removed because they are not used, and the home directory could be moved and the data here would be incoherent.

I wonder if we should do the same with the profile name and the creation date, not sure if we need them.
The creation date can be useful though to give an idea of when a profile was created when listing profiles.
The name is also useful, but could be problematic to have it in the metadata in case it is moved. Though we don't support manually handling files inside the data dir.

Reply:
As you mentioned, I would also remove User and Path.

Creation date it could be interesting to be kept to show in the profiles list view for the users in case they need compare which profile should be using (or deleting).

> The name is also useful, but could be problematic to have it in the metadata in case it is moved.

Currently, we don't support moving. Just copying a profile based on another profile. So it should not be a problem now that, based on users are not updating those files manually :sweat_smile:

Maybe raise an error if there are two profiles with the same name ? But that can be done just when running `elastic-package profiles list` , no?